src/table.js

```diff
@@ -355,6 +355,8 @@ function alignof(base = {}, data, columns) {
 }
 
 function type(data, column) {
+  // duck-type Arrow table date fields
+  if (String(data?.schema?.fields?.find?.(d => d?.name === column)).match(/<MILLISECOND>/)) return "date";
```

Comment:
Maybe a better fit:
```suggestion
  if (String(data?.schema?.fields?.find?.(d => d?.name === column)).endsWith("<MILLISECOND>") return "date";
```

I also wonder if the string representation is part of arrow's public API. Should we perhaps check against the typeId instead?

Reply:
I don't know… maybe… I tried to identify something that had a chance to be a bit stable, but it's a moving target.

Reply:
From what I recall the positive typeIds are public, and negative IDs are meant for internal use.